src/Authentication/Authentication/Microsoft.Graph.Authentication.csproj

```diff
@@ -22,7 +22,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Graph.Auth" Version="1.0.0-preview.5" />
-    <PackageReference Include="Microsoft.Graph.Core" Version="1.20.1" />
+    <PackageReference Include="Microsoft.Graph.Core" Version="1.24.0" />
```

Comment:
```suggestion
    <PackageReference Include="Microsoft.Graph.Core" Version="1.20.1 />
```
Bumping to 1.24.0 without setting the correct dependency versions results in missing dll in PS 5.1. 
![image](https://user-images.githubusercontent.com/7061532/105786309-a61eba80-5f31-11eb-915b-50659a9fa5e7.png)

Consider reverting back to 1.20.1 since I'll be updating this lib when I enable CAE.

Reply:
There is a bug fix in 1.24, regarding header usage that went out last year https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/pull/123.
We need the PR to be able to know what the response type is.

Reply:
Then we should update the dependencies to avoid the error above in PS 5.1.